[2021-11-15 10:22:21,287] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: covid_ingestion.ingest_to_dwh manual__2021-11-15T10:22:19.777718+00:00 [queued]>
[2021-11-15 10:22:21,313] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: covid_ingestion.ingest_to_dwh manual__2021-11-15T10:22:19.777718+00:00 [queued]>
[2021-11-15 10:22:21,313] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-15 10:22:21,313] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-15 10:22:21,313] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-15 10:22:21,328] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): ingest_to_dwh> on 2021-11-15 10:22:19.777718+00:00
[2021-11-15 10:22:21,333] {standard_task_runner.py:52} INFO - Started process 3901 to run task
[2021-11-15 10:22:21,336] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'covid_ingestion', 'ingest_to_dwh', 'manual__2021-11-15T10:22:19.777718+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/covid_ingestion.py', '--cfg-path', '/tmp/tmphsqkhjqj', '--error-file', '/tmp/tmpaem824mg']
[2021-11-15 10:22:21,337] {standard_task_runner.py:77} INFO - Job 69: Subtask ingest_to_dwh
[2021-11-15 10:22:21,388] {logging_mixin.py:109} INFO - Running <TaskInstance: covid_ingestion.ingest_to_dwh manual__2021-11-15T10:22:19.777718+00:00 [running]> on host 6f9154c92b39
[2021-11-15 10:22:21,445] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=covid_ingestion
AIRFLOW_CTX_TASK_ID=ingest_to_dwh
AIRFLOW_CTX_EXECUTION_DATE=2021-11-15T10:22:19.777718+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-15T10:22:19.777718+00:00
[2021-11-15 10:22:21,455] {base.py:79} INFO - Using connection to: id: postgres_dwh. Host: postgres-dwh, Port: 5432, Schema: None, Login: ***, Password: ***, extra: {}
[2021-11-15 10:22:21,459] {dbapi.py:225} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS pet (
            pet_id SERIAL PRIMARY KEY,
            name VARCHAR NOT NULL,
            pet_type VARCHAR NOT NULL,
            birth_date DATE NOT NULL,
            OWNER VARCHAR NOT NULL);
          , parameters: None
[2021-11-15 10:22:21,461] {postgres.py:71} INFO - NOTICE:  relation "pet" already exists, skipping

[2021-11-15 10:22:21,478] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=covid_ingestion, task_id=ingest_to_dwh, execution_date=20211115T102219, start_date=20211115T102221, end_date=20211115T102221
[2021-11-15 10:22:21,509] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-15 10:22:21,545] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
